portfolio/src/main/webapp/portfolio.html

```diff
@@ -47,12 +54,9 @@ <h3 class="header">PointMe</h3>
       <div id="facts">Wanna learn more about me?</div>
       <div id="generate-fact">Click <button onclick="getRandomFact()">here</button> for a random fun fact!</div>
     </footer>
-
-    <!-- Get Ionicons -->
-    <script src="https://unpkg.com/ionicons@5.0.0/dist/ionicons.js"></script>
```

Comment:
The Ionicons are only needed in index.html since I have the social media links only on that page, so I removed them in portfolio.html to remove overhead.